### DISCUSSION TITLE
How can I enable syntax highlighting in the text view?

### DISCUSSION THREAD

**Discussion by westhom** [josdejong/svelte-jsoneditor#403] [Q&A]
<img width="720" alt="Screenshot 2024-02-13 at 12 45 11" src="https://github.com/josdejong/svelte-jsoneditor/assets/921720/0c7af52f-4bb2-4847-aabc-ca28174d09dc">

That's what my editor looks like. The tree view has proper highlighting. Here's what my config looks like (inside a Vue SFC):

```js
editor = new JSONEditor({
    target: editor_div.value!,
    props: {
      statusBar: false,
      navigationBar: false,
      mode: Mode.text,
      onChange: contentChanged,
      content: {
        json: props.json
      }
    }
})
```

**Comment by josdejong** [comment]
How can we reproduce your issue? Can you create a minimal demo at https://jsbin.com or https://stackblitz.com/ ?

**Comment by josdejong** [comment]
Closing this issue due to a lack of response, please reopen if needed.
